### DISCUSSION TITLE
Anybody gets a step by step ssl certificate generation document to generate nebula usable ca ssl certificate？

### DISCUSSION THREAD

**Discussion by mo-avatar** [vesoft-inc/nebula#5579] [Q&A] [answered]
Any body hava a step by step ssl certificate generation document to generate usable ca ssl certificate which can be used by nebula。
The  ssl certificate  generation method which I find on internet seems all can’t work, it is really painful for new bee of openssl. Many thanks！

**Comment by wey-gu** [comment, marked as answer]
recommend to try with mkcert: https://github.com/FiloSottile/mkcert
